File: core/src/main/java/at/shockbytes/corey/common/core/util/WorkoutNameComparator.java
package at.shockbytes.corey.common.core.util;

import java.util.Comparator;

import at.shockbytes.corey.common.core.workout.model.Workout;

/**
 * @author Martin Macheiner
 *         Date: 29.03.2017.
 */

public class WorkoutNameComparator implements Comparator<Workout> {

    @Override
    public int compare(Workout workout, Workout t1) {
        return workout.getDisplayableName().compareToIgnoreCase(t1.getDisplayableName());
    }
}

File: core/src/main/java/at/shockbytes/corey/common/core/util/WorkoutNameComparator.kt
package at.shockbytes.corey.common.core.util

import at.shockbytes.corey.common.core.workout.model.Workout
import java.util.Comparator

/**
 * Author:  Martin Macheiner
 * Date:    29.03.2017
 */
class WorkoutNameComparator : Comparator<Workout> {

    override fun compare(workout: Workout, t1: Workout): Int {
        return workout.displayableName.compareTo(t1.displayableName, ignoreCase = true)
    }
}
